lighthouse-core/audits/is-on-https.js

```diff
@@ -72,6 +72,11 @@ class HTTPS extends Audit {
       extendedInfo: {
         formatter: Formatter.SUPPORTED_FORMATS.URL_LIST,
         value: insecureRecords
+      },
+      details: {
```

Comment:
I'd be in favor of renaming `extendedInfo` to `details`. We don't need both and `extendedInfo` is really just a details object with a specified formatter.

When I was looking over this code, I also found the generalization of `_renderDetails` to be premature. Not sure we need recursive rendering of details just yet (`_renderDetails` calls `_renderBlock|List|Text` which in turn, call `_renderDetails`). That's nice if we ever have nests of `<details>` but to date, haven't need that.